check/test.go

```diff
@@ -230,7 +229,7 @@ func (ts *tests) execute(s string) *testOutput {
 	}
 
 	finalOutput.testResult = result
-	finalOutput.actualResult = res[0].actualResult
+	finalOutput.actualResult = s
```

Comment:
This looks strange to me - isn't `s` the audit command to be executed?

Reply:
it's the output of the audit command

Reply:
Ah, OK... i can't help thinking that the name `execute()` is pretty bad for these functions - more like `checkResults()`. 

On the occasions where we call `failTestItem()` the string is currently being set to an error message explaining what's wrong (e.g. bad YAML). I think with this change we are going to lose those messages as they're going to get overwritten by the original string. Wdyt?

Reply:
True, fixing it now